src/init/autoApplyCustomizations.ts

```diff
@@ -6,7 +6,7 @@ export function manageAutoApplyCustomizations(
   userConfig: IVSIcons,
   applyCustomizationCommand: () => void): void {
   if (!isNewVersion) { return; }
-  const propObj = packageJson.contributes.configuration.properties || {};
+  const propObj: any = packageJson.contributes.configuration.properties as object;
```

Comment:
Now, this is a case of a functionality not been implemented by others. `Typescript` in 2.2.x introduced the use of `object` versus `Object`. When I used that, `tslint` complained that `popObj` was not marked as an `Object` (although technically it is). So I was forced to use `any` until `tslint` fixes the rule.